custom-vu/src/main/kotlin/jces1209/vu/utils/boards/CsvBoardsReader.kt

```diff
@@ -0,0 +1,48 @@
+package jces1209.vu.utils.boards
+
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import java.lang.IllegalStateException
+import java.net.URI
+
+object CsvBoardsReader {
+
+    var csvContent: MutableList<CsvBoard>?
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+    private const val csvPropertyFile = "BoardUsageFrequency.csv"
+
+    init {
+        csvContent = readBoardsFromCsv()
```

Comment:
csvContent  is redundant here
this is a reader but not a holder

Let's just use readBoardsFromCsv(csvPropertyFile: String) in BoardsFrequencyManager

Reply:
Fixed according to the comment.